### PR TITLE
Add dev/pkg for MP-3014-1100

### PR DESCRIPTION
**Pull request by dbrgn**
Datasheet:

https://download.luminus.com/datasheets/Luminus_MP3014_1100_Datasheet.pdf

Packages (reflow and handsoldering):

Gerber output:

Package:

Device:

### CODE REVIEW

**Comment by ubruhin** [comment]
Very nice! Only two things:
- The package category does not seem to exist in our libraries. Maybe add a new category "Luminus Chip LED" as a subcategory of "Diode"?
- Maybe prefixing package and device names with "Luminus "? The library name is not considered as namespace, so probably it makes sense to put the manufacturer into package/device names (of course only for things which are *really* manufacturer specific, which this LED seems to be). I already did this in the Isabellenhütte library.

**Comment by dbrgn** [comment]
> The package category does not seem to exist in our libraries.

Hmm, I chose "Custom" and then forgot about it, probably I created that in another library than Base.

> Maybe add a new category "Luminus Chip LED" as a subcategory of "Diode"?

What about a generic "LED" category below "Diodes"? Maybe with "Chip LED" as subcategory? (Would 5050 LEDs also be Chip LEDs?)

> Maybe prefixing package and device names with "Luminus "? The library name is not considered as namespace, so probably it makes sense to put the manufacturer into package/device names (of course only for things which are really manufacturer specific, which this LED seems to be).

Ah, that's a good point. I initially considered the library to be a namespace, but of course we don't always group by manufacturer.

**Comment by ubruhin** [comment]
> What about a generic "LED" category below "Diodes"? Maybe with "Chip LED" as subcategory?

Would also be OK, but I wouldn't create too many subcategories. Another option would be to add manufacturer specific categories, for example:
- Diode
  - Diode Outline (DO)
  - Small-Outline Diode (SOD)
- [...]
- Manufacturer-Specific
  - Luminus
  - Microchip
  - Isabellenhütte

What do you think?

**Comment by dbrgn** [comment]
I think I'd stick with purpose-specific terminology for the categories, not grouped by manufacturer.

For now, I'd be fine with just adding a LED category to the Base library without any subcategories.

**Comment by dbrgn** [comment]
Hm, on second thought, the current footprints are grouped by shape, not by purpose. The "Diode" Category stands for a certain package (the DO and SOD packages), if diodes have other packages then they won't be put there.

Thus, I'd add a "Manufacturer-Specific" category to the Base library that can be extended with subcategories by other libraries.

**Comment by dbrgn** [comment]
Updated with new pkgcat (placed below the one added in https://github.com/LibrePCB-Libraries/LibrePCB_Base.lplib/pull/7) and rebased/squashed.

File: pkg/e1eb50b1-28ba-4aa8-932d-d8addb5ce9e7/package.lp [hunk 1]
```diff
@@ -0,0 +1,190 @@
+(librepcb_package e1eb50b1-28ba-4aa8-932d-d8addb5ce9e7
+ (name "Luminus MP3014")
```
Comment (ubruhin) [inline pkg/e1eb50b1-28ba-4aa8-932d-d8addb5ce9e7/package.lp]: I guess there should be a dash(?):

```suggestion
 (name "Luminus MP-3014")
```

Comment (dbrgn) [reply pkg/e1eb50b1-28ba-4aa8-932d-d8addb5ce9e7/package.lp]: Yes you're right. Fixed.

**Review by ubruhin — APPROVED** [review]

**Pull request merged**